FreeRTOS/FreeRTOS
# prepare for datecodes, don't update versions for submodules

Script for updating freertos version in file headers. Now using date codes.

Test Steps
-----------
Run `-h` on the script. Update some temp freertos repo with the version number of your liking, and verify the version numbers were updated.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
